app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -532,4 +534,20 @@ public void onFailure(Call<ResponseBody> call, Throwable t) {
                 });
     }
 
+    /**
+     * Sets maximum message content editing limit in seconds
+     *
+     * @param seconds seconds to be set as maximum limit
+     */
+    private void setMessageContentEditLimitSeconds(int seconds) {
+        SharedPreferences preferences = getSettings();
+
+        //Checks id maximum content edit limit is already saved or not
+        if (!preferences.getBoolean(Constants.IS_MAXIMUM_CONTENT_EDIT_LIMIT_SAVED, false)) {
```

Comment:
yes as Kunal said, this is the "event" we receive from the server immediately when a realm configuration property is changed. So for real time sync(during the run time of app) you can listen for this event in [processEvents](https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/networking/AsyncGetEvents.java#L304) by making a wrapper class. Example [messageWrapper](https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/networking/response/events/MessageWrapper.java).